### PR TITLE
[kube-prometheus-stack] make nodeexporter & config-reloaders rules configurable

### PR DESCRIPTION
**Pull request by monotek**
Signed-off-by: André Bauer <andre.bauer@staffbase.com>

#### What this PR does / why we need it:

* makes nodeexporter recording and altering rules configurable separately
* makes config reloaders alerts configurable
* sorted rules in python script alphabetically
* removed the following values (and according python code) as they seem to have no effect in defaultRules.rules:
  * kubeApiserver
  * kubeApiserverError
  * kubePrometheusNodeAlerting
  * kubernetesAbsent
  * time
  * added runbookurl config

#### Which issue this PR fixes
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
  - fixes #

#### Checklist

- [x] [DCO](https://github.com/prometheus-community/helm-charts/blob/main/CONTRIBUTING.md#sign-off-your-work) signed
- [x] Chart Version bumped
- [x] Title of the PR starts with chart name (e.g. `[prometheus-couchdb-exporter]`)

### CODE REVIEW

**Comment by mrueg** [comment]
LGTM, It would be nice to get https://github.com/prometheus-community/helm-charts/pull/1626 in before

**Comment by monotek** [comment]
As we cant update the branch of @hairmare in #1626 I could add these changes to this pr too.

File: charts/kube-prometheus-stack/README.md [hunk 1]
```diff
@@ -83,6 +83,13 @@ _See [helm upgrade](https://helm.sh/docs/helm/helm_upgrade/) for command documen
 
 A major chart version change (like v1.2.3 -> v2.0.0) indicates that there is an incompatible breaking change needing manual actions.
 
+### From 26.x to 27.x
```
Comment (mrueg) [inline charts/kube-prometheus-stack/README.md:86]: Can you add a note that this adds the runbook url back?

Comment (monotek) [reply charts/kube-prometheus-stack/README.md:86]: Didn't put it in because no breaking change... But yes :)

**Comment by monotek** [comment]
I've added a note to the Readme. Used `runbookUrl: "https://runbooks.prometheus-operator.dev/runbooks"` as default value. The ending slas is in the template now.

While we're on a major version anyway. Whats your opinion on: https://github.com/prometheus-community/helm-charts/issues/1631#issuecomment-1002644013 ?

Should we set a fixed serviceMonitorSelector here, to get kube-state-metrics working out of the box?

**Comment by Xtigyro** [comment]
@monotek Yeah - I think those should be enabled by default.

**Comment by monotek** [comment]
I guess we also could just add the "release" label to the kube-state-metrics chart. Imho this would have the least user impact.

**Review by mrueg — DISMISSED** [review]

**Pull request merged**